packages/flutter_tools/lib/src/commands/build_ipa.dart

```diff
@@ -0,0 +1,154 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/file.dart';
+import 'package:meta/meta.dart';
+import 'package:process/process.dart';
+
+import '../application_package.dart';
+import '../base/common.dart';
+import '../base/logger.dart';
+import '../base/platform.dart';
+import '../base/process.dart';
+import '../build_info.dart';
+import '../convert.dart';
+import '../globals.dart' as globals;
+import '../ios/xcodeproj.dart';
+import '../runner/flutter_command.dart' show FlutterCommandResult;
+import 'build.dart';
+
+/// Builds an iOS .ipa package for App Store submission.
+///
+/// A wrapper around "xcodebuild -exportArchive". Can only be run on a macOS host.
+class BuildIPACommand extends BuildSubCommand {
+  BuildIPACommand({
+    @visibleForTesting Platform platform,
+    @visibleForTesting FileSystem fileSystem,
+    @visibleForTesting Logger logger,
+    @visibleForTesting ProcessManager processManager,
+    @visibleForTesting XcodeProjectInterpreter xcodeProjectInterpreter,
+  })  : _platform = platform,
+        _xcodeProjectInterpreter = xcodeProjectInterpreter,
+        _fileSystem = fileSystem,
+        _logger = logger,
+        _processUtils =
+            ProcessUtils(processManager: processManager, logger: logger) {
+    argParser.addOption(
+      'export-options-plist',
+      valueHelp: 'ExportOptions.plist',
+      help:
+          'Path to the "-exportOptionsPlist" file passed to Xcode. See "man xcodebuild" for available keys.',
```

Comment:
For first timers (which we will have), we might need to link to a Flutter page to describe what it is. Since we'll certainly have Flutter developers shipping for the first time to the app store without having shipped an ios app in the past.

Feel free to come back in a later PR too instead of leaving a placeholder.

Reply:
For sure, we'll need to point to some docs.  But we can pretty much just point to Apple's docs since this command is a really thin wrapper and doesn't process any plist values.

Reply:
Added a TODO to update the new option help text.